src/app/components/challenge/challengeleaderboard/challengeleaderboard.component.ts

```diff
@@ -245,6 +260,15 @@ export class ChallengeleaderboardComponent implements OnInit, AfterViewInit {
       SELF.selectedPhaseSplit = phaseSplit;
       if (SELF.selectedPhaseSplit) {
         SELF.fetchLeaderboard(SELF.selectedPhaseSplit['id']);
+        if (SELF.phaseSplits[SELF.selectedPhaseSplit['id']].hasOwnProperty('show_leaderboard_by_latest_submission')) {
+          const toggleByLatest = SELF.phaseSplits[SELF.selectedPhaseSplit['id']].hasOwnProperty('show_leaderboard_by_latest_submission');
+
+          if (toggleByLatest === true) {
+            SELF.showLeaderboardByLatest = true;
+          } else {
+             SELF.showLeaderboardByLatest = false;
```

Comment:
@drepram Here you should directly set `SELF.showLeaderboardByLatest = SELF.selectedPhaseSplit.show_leaderboard_by_latest_submission`

Reply:
Can I make a ternary operator that would be a fallback if there is no value?

```javascript
SELF.showLeaderboardByLatest = ('selectedPhaseSplit.show_leaderboard_by_latest_submission' in self ? selectedPhaseSplit.show_leaderboard_by_latest_submission : true)
```

^ this is for the true case, for the false case it would be different

Reply:
`SELF.showLeaderboardByLatest = SELF.selectedPhaseSplit.show_leaderboard_by_latest_submission` This is more cleaner I think. @drepram